frb_dart/lib/src/manual_impl/_io.dart

```diff
@@ -12,10 +12,24 @@ List<dynamic> wireSyncRust2DartDcoIntoDart(WireSyncRust2DartDco syncReturn) =>
     dartCObjectIntoDart(syncReturn.ref);
 
 /// {@macro flutter_rust_bridge.only_for_generated_code}
-int dcoDecodeI64(int raw) => raw;
+int dcoDecodeI64(dynamic raw) {
+  if (raw is int) {
+    return raw;
+  } else {
+    throw Exception(
+        'dcoDecodeI64 unexpected type=${raw.runtimeType} value=$raw');
+  }
+}
 
 /// {@macro flutter_rust_bridge.only_for_generated_code}
-BigInt dcoDecodeU64(int raw) => BigInt.from(raw).toUnsigned(64);
+BigInt dcoDecodeU64(dynamic raw) {
+  if (raw is num) {
+    return BigInt.from(raw).toUnsigned(64);
+  } else {
+    throw Exception(
+        'dcoDecodeU64 unexpected type=${raw.runtimeType} value=$raw');
+  }
+}
```

Comment:
Hmm, for these two functions, maybe we can keep the original version (or add a `as` cast)